abuild.in

```diff
@@ -695,6 +695,14 @@ postcheck() {
 			&& ! is_openrc_pkg "$name"; then
 		warning "Found OpenRC directory (/etc/conf.d or /etc/init.d) but name doesn't end with -openrc"
 	fi
+	# look for static archives
+	if ! is_static_pkg;  then
+		for i in $(find "$dir"/lib "$dir"/usr/lib -name '*.a' 2>/dev/null); do
+			case "$(file -bi $i)" in
```

Comment:
POSIX `file` functions very differently to the `file` provided by the `file` package.
Namely: the `-i` argument actually avoids classification, and does not have the `-b` argument.
abuild does not explicitly depend on the `file` package (though build-base and alpine-sdk do).

I think this check can be skipped altogether, since it seems to be ultimately implementation dependent, and the consequence of getting it wrong the other way is just a warning being printed.

Reply:
The check has been removed.